### ISSUE TITLE
Wait for Keptn Event: The wait for event does not wait long enough

### ISSUE THREAD

**Issue by robertjahn** [keptn-sandbox/keptn-azure-devops-extension#2]
Looks like it the add is just trying once to get eval complete event. Needs to wait longer.

It would be nice to have this configurable with a default.

![image](https://user-images.githubusercontent.com/24907089/85057521-db2e8780-b16e-11ea-9b18-702acd043ef2.png)

**Comment by heydenb** [comment]
Hey Rob,

I checked again and it should wait for 2 minutes now.
Would it be possible for you to send that https://keptn-endpoint/v1/event?type=sh.keptn.events.evaluation-done&keptnContext=%keptnContext% yourself via the swagger-ui to see what you get back?
I check on a http 500 and the message must be: "No Keptn sh.keptn.events.evaluation-done event found for context"
In that case I wait for another 10 seconds. And this is repeated 12 times.

**Comment by robertjahn** [comment]
Still seems to happen.  But see this log it says version 1.3

![image](https://user-images.githubusercontent.com/24907089/85893337-b49ccc00-b7c0-11ea-88f8-ca3611f4a26f.png)

yet I see 1.8 in org settings

![image](https://user-images.githubusercontent.com/24907089/85893374-c4b4ab80-b7c0-11ea-8b3f-b5ac57ed88c5.png)

**Comment by robertjahn** [comment]
The task version needs to be updated.

https://github.com/keptn-sandbox/keptn-azure-devops-extension/blob/master/WaitForKeptnEventTask/task.json#L16

![image](https://user-images.githubusercontent.com/24907089/85893503-fc235800-b7c0-11ea-95f4-3dc4824e7a31.png)


You have same version number labeling issue in the other task.json files too.  for example
https://github.com/keptn-sandbox/keptn-azure-devops-extension/blob/master/PrepareKeptnEnvTask/task.json#L16

**Comment by heydenb** [comment]
Tasks don't need to follow the same version as the extension itself. The task version was 1.2 and is now 1.3, so that should not be the issue. Maybe we schedule a meeting to sort this out.

**Comment by robertjahn** [comment]
Thats fine about the version number in the task.  I understand it can be tracked differently that the app.  I should not have mixed the two topics.  But this version number in the name I think we should still do and we can discuss that in this issue: https://github.com/keptn-sandbox/keptn-azure-devops-extension/issues/10

As to this wait time problem, I did more testing and I found that in my Keptn environment that is takes longer than two minutes to get the evaluation results.  I was able to get the results two ways 1) adding a "sleep 60" step after the send event and the wait for event.  2)  by having one wait event test with continue on error and then having a second wait for event.  The first one got the no results found and the second wait for event got the results.

Will you:

1) have this configurable field in the task with a default of 120.  Also, you ought to display the value used in the task output. like "using timeOut 300 seconds"

2) add to output "Waiting, on the Loop XXX" in the the loop (https://github.com/keptn-sandbox/keptn-azure-devops-extension/blob/master/WaitForKeptnEventTask/index.ts#L114). This way we know it is processing and how many loops it took.

3) adjust the output to not show the count and "not-found" status.  Looks like this now....
![image](https://user-images.githubusercontent.com/24907089/86069794-ede76d00-ba48-11ea-99f8-7e5864afebf6.png)

Have it instead for the not found show a user friendly message to something like "Keptn evaluation failed.  Keptn context not found"
